static/bureaux.json

```diff
@@ -0,0 +1,1044 @@
+{
```

Comment:
Consider removing the `bureaux` attribute and just put the data directly to avoid using `bureauxJSON.bureaux` and just use `bureauxJSON` when importing the file.

```
{
  "bureaux":[{emoji:"",nom:"",...}]
}
```

To :

```
[{emoji:"",nom:"",...}]
```